Makefile

```diff
@@ -176,6 +176,17 @@ build-test:
 	@test/build/test-configure.sh
 
 
+.PHONY: mango-on-travis
```

Comment:
Why not just call it `test-mango`? I can see using this not just in Travis.

If we're not going to migrate the mango tests to etap or javascript we might as well just add this as a new target for `make check`. Then, the `.travis.yml` file won't change at all and we simply extend our normal test run slightly.